public/app/features/alerting/components/AlertingQueryPreview.tsx

```diff
@@ -26,12 +28,11 @@ interface Props {
   onRunQueries: () => void;
 }
 
-export const AlertingQueryPreview: FC<Props> = ({ getInstances, onRunQueries, onTest, queryRunner, queries }) => {
+export const AlertingQueryPreview: FC<Props> = ({ getInstances, onRunQueries, onTest, queries, queryRunner }) => {
```

Comment:
After seeing how @mckn solved passing the `queryRunner` to components with a ContextProvider and hooks I think you should look into this approach and move querRunner out of the Redux state. But all this can be done in a follow up PR.

Reply:
Yeah, we will redo this in another PR